### DISCUSSION TITLE
Can't find Channels in M3u lik

### DISCUSSION THREAD

**Discussion by yassineatik** [iptv-org/community#328] [Q&A] [answered]
Hello guys , i hope you're all are doing well
I actually want to watch bein sports in this m3u link https://iptv-org.github.io/iptv/index.m3u , but i could not find it
But in the EPG link , i found BeIn sports is already in the guides 
I just wanna know if all the channels in the guides are in the m3u link ? or it's a different thing?

**Comment by BellezaEmporium** [comment, marked as answer]
As we've told many times recently, what you see in iptv-org.github.io and the EPGs are **not** representative of what we have in the playlists.

And those channels are banned from getting inside our playlists as they're DMCAd.
